main.go

```diff
@@ -17,24 +17,25 @@ limitations under the License.
 package main
 
 import (
-	"flag"
-	"github.com/aws/cluster-api-provider-cloudstack/pkg/cloud"
+	"fmt"
 	"os"
-	clusterv1 "sigs.k8s.io/cluster-api/api/v1alpha3"
 	"strings"
 
-	// Import all Kubernetes client auth plugins (e.g. Azure, GCP, OIDC, etc.)
-	// to ensure that exec-entrypoint and run can make use of them.
+	"github.com/aws/cluster-api-provider-cloudstack/pkg/cloud"
+	flag "github.com/spf13/pflag"
```

Comment:
pflag is a 'drop in' replacement for the builtin flag that supports posix compatible flags.

So --metrics-addr instead of -metrics-addr.